### PR TITLE
f-footer@v4.26.0 - Add Percy Visual Regression Tests

### PR DESCRIPTION
**Pull request by siggerzz**
v4.26.0
------------------------------
*June 15, 2021*

### Added
- Added Percy Visual Regression tests

### CODE REVIEW

**Review by JoshuaNg2332 — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Comment by DamianMullins** [comment]
@siggerzz Looks like you need to add a version bump and changelog entry for the root, f-button, f-checkout, f-cookie-banner components?

**Review by JoshuaNg2332 — DISMISSED** [review]

File: packages/components/organisms/f-checkout/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "0.141.0",
+  "version": "0.141.1",
```
Comment (DamianMullins) [inline packages/components/organisms/f-checkout/package.json]: Was there any reason this was a patch bump and the others were minor version bumps?

Comment (siggerzz) [reply packages/components/organisms/f-checkout/package.json]: I had to fix some visual regression tests in this one 👍🏼

Comment (DamianMullins) [reply packages/components/organisms/f-checkout/package.json]: I'd say you need to bump the version consistently as this project contained the same changes and more compared to the other package yet they had a minor version bump with fewer changes?

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "@justeat/eslint-config-fozzie": "3.4.1",
     "@justeat/fozzie": "5.0.0-beta.8",
     "@justeat/stylelint-config-fozzie": "2.2.0",
-    "@percy/cli": "1.0.0-beta.48",
+    "@percy/cli": "1.0.0-beta.52",
```
Comment (DamianMullins) [inline package.json:39]: Looks like the danger checks require a version bump on here as well.

**Review by DamianMullins — DISMISSED** [review]

**Review by JoshuaNg2332 — DISMISSED** [review]

**Review by DamianMullins — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

**Pull request merged**